mbf_abstract_nav/include/mbf_abstract_nav/abstract_execution_base.h

```diff
@@ -39,11 +39,12 @@
 #ifndef MBF_ABSTRACT_NAV__ABSTRACT_EXECUTION_BASE_H_
 #define MBF_ABSTRACT_NAV__ABSTRACT_EXECUTION_BASE_H_
 
-#include <mbf_abstract_nav/MoveBaseFlexConfig.h>
 #include <boost/thread.hpp>
 #include <boost/chrono/duration.hpp>
 #include <boost/chrono/thread_clock.hpp>
 
+#include <mbf_abstract_nav/MoveBaseFlexConfig.h>
```

Comment:
`#include "mbf_abstract_nav/MoveBaseFlexConfig.h"`

prefers the workspace version more than the apt package